metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg_profiler.py

```diff
@@ -208,17 +222,18 @@ def _renderValue(
             return None
 
     @staticmethod
-    def _is_numeric_type(type: Type) -> bool:
```

Comment:
Should we add `BooleanType` here as well? I would even consider just profiling all the types. In Iceberg, by default the strings are truncated at a length of 16 characters, so I think it would also be interesting to add just profile everything.

Reply:
That's what I had at the beginning (all types were profiled), but then I wonder if there was value in profiling truncated strings.  I wonder if it could be misleading for people thinking a specific value is the min or max, and the value doesn't even exist if you do a `SELECT * FROM x WHERE col = <min|max>`